BigBaseV2/src/backend/looped/self/clean_player.cpp

```diff
@@ -0,0 +1,18 @@
+#include "backend/looped/looped.hpp"
+#include "natives.hpp"
+
+namespace big
+{
+	void looped::self_clean_player()
+	{
+		auto const player_ped = PLAYER::PLAYER_PED_ID();
+
+		if (g->self.clean_player) {
+			PED::CLEAR_PED_BLOOD_DAMAGE(player_ped);
+			PED::CLEAR_PED_WETNESS(player_ped);
+			PED::CLEAR_PED_ENV_DIRT(player_ped);
+			PED::RESET_PED_VISIBLE_DAMAGE(player_ped);
```

Comment:
Make these calls a function, you're using these same natives again in a looped feature.

util/entity.hpp
```cpp
inline void clean_ped(Ped ped)
{
	PED::CLEAR_PED_BLOOD_DAMAGE(PLAYER::PLAYER_PED_ID());
	PED::CLEAR_PED_WETNESS(PLAYER::PLAYER_PED_ID());
	PED::CLEAR_PED_ENV_DIRT(PLAYER::PLAYER_PED_ID());
	PED::RESET_PED_VISIBLE_DAMAGE(PLAYER::PLAYER_PED_ID());
}
```